blocks/product-details/product-details.js

```diff
@@ -146,10 +151,16 @@ export default async function decorate(block) {
           const values = pdpApi.getProductConfigurationValues();
           const valid = pdpApi.isProductConfigurationValid();
 
-          // add the product to the cart
           if (valid) {
-            const { addProductsToCart } = await import('@dropins/storefront-cart/api.js');
-            await addProductsToCart([{ ...values }]);
+            if (isEditMode) {
+              const { updateProductsFromCart, addProductsToCart } = await import('@dropins/storefront-cart/api.js');
+              await updateProductsFromCart([{ uid: itemUid, quantity: 0 }]);
+              await addProductsToCart([{ ...values }]);
+              window.location.href = rootLink('/cart');
+            } else {
+              const { addProductsToCart } = await import('@dropins/storefront-cart/api.js');
+              await addProductsToCart([{ ...values }]);
+            }
```

Comment:
Since the update or add to cart API and logic was already implemented in PR https://github.com/hlxsites/aem-boilerplate-commerce/pull/419, we could remove this. Thanks for adding this though. 
```suggestion
```